homeassistant/components/asuswrt/router.py

```diff
@@ -179,11 +181,56 @@ def __init__(self, hass: HomeAssistant, entry: ConfigEntry) -> None:
             self.hass, dict(self._entry.data), self._options
         )
 
+    def _migrate_entities_unique_id(self) -> None:
```

Comment:
This has nothing to do with a protocol. This can be split out into a different PR.

Reply:
Moved to separate PR